### ISSUE TITLE
Pseudo-Huber loss and weighting

### ISSUE THREAD

**Issue by nirgoren** [Luo-Yihong/TDM#13]
Hello,

In the paper the constant `c` used in the Pseudo-Huber loss is `c = 0.00054 * d^0.5`, yet in the demo a constant `c=0.001` is used. Additionally in the demo there is an additional weighting factor being used that is not mentioned in the paper:

`weighting_factor = torch.abs(model_latents.double() - sd_latents_cfg.double() ).mean(dim=[1, 2, 3], keepdim=True).detach()`

Can you explain these differences in the implementation, the motivation behind them and how critical they are to the success of the method?

Thank you!

**Comment by Luo-Yihong** [comment]
We found that Huber loss is relatively insensitive to this constant, so we use a default value of 1e-3 in the public repository. 

The weighting factor is a widely used technique in recent score distillation. In my experience, removing it still allows Huber loss to work well, but MSE loss suffers from notable performance degradation.

**Comment by nirgoren** [comment]
Thank you for your answer!
